src/binaryen.cpp

```diff
@@ -623,6 +623,10 @@ void BinaryenEngine::verifyContract(wasm::Module & module)
   };
 
   for (auto const& import: module.functions) {
+
+    if (import->body)  // Skip defined functions.
+      continue;
```

Comment:
Good catch, why did this work beforehand?

Reply:
Previously it imported and defined functions were distinct collections. Binaryen has changed that and now all functions are in the same vector.
I'm not sure this is best way to check if a function is imported, but I have not found anything better so far.

Reply:
Looking at the commits individually explains it. Originally the code used `module.imports`. Therefore I think it would be better to squash `Binaryen: Only check imported functions in contract verification` into the first commit.